src/django/api/moderation_event_actions/creation/location_contribution/processors/permission_processor.py

```
@@ -12,53 +12,117 @@
 class PermissionProcessor(ContributionProcessor):
 
+    TYPE_VALIDATORDS = {
+        'int': lambda value: isinstance(value, int)
+        and not isinstance(value, bool),
+        'float': lambda value: isinstance(value, float)
+        and not isinstance(value, bool),
+        'string': lambda value: isinstance(value, str),
+        'object': lambda value: isinstance(
+            value, (dict, list)
+        ),
+    }
```

Ruff (0.12.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)